sync/firewall/config.go

```diff
@@ -1,16 +1,42 @@
 package firewall
 
+import (
+	_ "embed"
+	"encoding/json"
+)
+
+//go:embed black_list.json
+var _defaultBlackListIPs []byte
+
 type Config struct {
-	Enabled bool `toml:"enable"`
+	Enabled      bool     `toml:"enable"`
+	BlackListIPs []string `toml:"blacklist_ips"`
+}
+
+type defaultBlackListIPs struct {
```

Comment:
Why we have defaulted here?

Reply:
This Default append to configuration list, some time we find faulty similar now we want banned on patch version.

Same now.